pom.xml

```diff
@@ -20,7 +20,7 @@
   <modelVersion>4.0.0</modelVersion>
   <groupId>io.fabric8</groupId>
   <artifactId>elasticsearch-cloud-kubernetes</artifactId>
-  <version>5.2.1-SNAPSHOT</version>
+  <version>5.2.2</version>
```

Comment:
Can you leave this as a SNAPSHOT version please (perhaps 5.2.2-SNAPSHOT)? This will be updated when I do the Maven release.